assets/css/project/breadcrumb.less

```diff
@@ -0,0 +1,48 @@
+/* Breadcrumb
+-------------------------------------------------- */
+
+.source_breadcrumb {
+    display: block;
+    vertical-align: 1px;
+    background: #f1f2f3;
+    margin: 0 -25px;
+    padding: 10px 25px;
+
+    white-space: nowrap;
+    overflow: hidden;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+}
+
+.source_breadcrumb_lst {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.source_breadcrumb_lst > * {
+    display: inline-block;
+    margin: 0 10px;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    line-height: 1.5;
+}
+.source_breadcrumb_i {
+    &:first-child {
+        margin-left: 0;
+    }
+    &:last-child {
+        margin-right: 0;
+    }
+}
+.source_breadcrumb_s {
```

Comment:
Would be really cool, if you could define separator as pseudo in `.source_breadcrumb_i`, to no create new list item in the dom.